src/mail.sh

```diff
@@ -8,6 +8,10 @@ include "$KW_LIB_DIR/kw_config_loader.sh"
 # Hash containing user options
 declare -gA options_values
 
+declare -a essential_config_options=('user.name' 'user.email' 'fail' # for testing only
+  'sendemail.smtpserver' 'sendemail.smtpuser' 'sendemail.smtpserverport')
+declare -a recommended_config_options=('sendemail.smtpencryption' 'sendemail.smtppass')
```

Comment:
I think it is worth adding a more detailed comment about these two arrays on top of the essential_config_options. Additionally, update the name to something that portraits the idea that those are used for tests only.

Reply:
I used to have them inside the verify function, but decided to move them to the top so they could be used for the interactive and setup functions as well. The comment pertains only to the `'fail'` entry, which is there temporarily to force a failure. Will add descriptive comments.